responses.py

```diff
@@ -28,7 +28,7 @@
 try:
     from requests.packages.urllib3.connection import HTTPHeaderDict
 except ImportError:
-    from urllib3.connection import HTTPHeaderDict
+    from urllib3._collections import HTTPHeaderDict
```

Comment:
I don't know about this. My understanding is that modules with `_` prefixes are internal and shouldn't be used by consumers of a package. I'm going to open an issue in urllib3 about the removed import causing a compatibility break.

Reply:
I changed the patch to import `HTTPHeaderDict` from  `urllib3.response` based on https://github.com/urllib3/urllib3/issues/2045#issuecomment-725497199